tank/test_models.py

```diff
@@ -363,7 +363,7 @@ def test_module(self, dynamic, device, config):
             )
         if config["model_name"] == "mobilenet_v3_small" and device in [
             "metal",
-            "vulkan",
+            "vulkan" "rocm",
```

Comment:
I think you. need a comma

Reply:
forgot to remove that, didn't want to have duplicate xfails. thanks